test/e2e/upgrades/statefulset.go

```diff
@@ -35,6 +36,17 @@ type StatefulSetUpgradeTest struct {
 
 func (StatefulSetUpgradeTest) Name() string { return "statefulset-upgrade" }
 
+func (StatefulSetUpgradeTest) SkipVersions(versions ...version.Version) bool {
+	minVersion := version.MustParseSemantic("1.5.0")
```

Comment:
Not really since it's only run once per StatefulSetUpgrade test... which should be only once.